src/main/scala/org/broadinstitute/dsde/firecloud/utils/TSVFormatter.scala

```diff
@@ -2,14 +2,34 @@ package org.broadinstitute.dsde.firecloud.utils
 
 import org.broadinstitute.dsde.firecloud.core.GetEntitiesWithType.EntityWithType
 import org.broadinstitute.dsde.firecloud.model.ModelSchema
-import spray.json.JsValue
 
 import scala.collection.immutable
 import scala.util.Try
 
+import spray.json.DefaultJsonProtocol._
+import spray.json.JsValue
+
 object TSVFormatter {
 
-  def makeTsvString(entities: List[EntityWithType], entityType: String) = {
+  def makeMembershipTsvString(entities: List[EntityWithType], entityType: String): String = {
+    val headers: immutable.IndexedSeq[String] = immutable.IndexedSeq("membership:" + entityType + "_id", entityType.replace("_set", "_id"))
+    val rows: List[IndexedSeq[String]] = entities filter { _.entityType == entityType } flatMap { entity =>
+      entity.attributes.getOrElse(Map.empty) flatMap { m =>
+        m._2.convertTo[List[JsValue]] map { jsValue =>
```

Comment:
The json attributes in a membership object (i.e. pair_set, sample_set, participant_set) represent rows as opposed to the root types, where the attributes represent columns. So, for each set attribute, we need to get all of the elements out of it and turn each one into a row. Here, `m._2` is the attribute list. We unmarshal that into a list and then work with each value. 

Here is a pair_set example:
```
[{
  "name": "pair_set_1",
  "entityType": "pair_set",
  "attributes": {
    "pairs": [{
      "entityType": "pair",
      "entityName": "1"
    }, {
      "entityType": "pair",
      "entityName": "2"
    }]
  }
}]
```